config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -32,8 +32,8 @@ spec:
 
     ### Pre-requisites
     * A Red Hat OpenShift subscription
-    * OCP 4.7 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
-    * [vSphere prequisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-prerequisites.md)
+    * OCP 4.8 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
+    * [vSphere prequisites](https://docs.openshift.com/container-platform/latest/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#preparing-vsphere-for-windows-containers)
```

Comment:
Maybe this should link to the 4.8 docs since this is a 4.8 release

Reply:
since there are no 4.8 OpensShift docs yet, I have pointed to latest since that should point to 4.8 after 4.8 GA.